packages/lab/src/progress/CircularProgress/CircularProgress.tsx

```diff
@@ -190,42 +147,38 @@ export const CircularProgress = forwardRef<
   return (
     <div
       className={clsx(className, "saltCircularProgress", {
-        [withBaseName("small")]: size === "small",
-        [withBaseName("medium")]: size === "medium",
-        [withBaseName("large")]: size === "large",
         [withBaseName("disabled")]: disabled,
       })}
       data-testid="circular-progress"
       ref={ref}
       role="progressbar"
       aria-label={ariaLabel}
-      aria-valuemax={MAX}
-      aria-valuemin={MIN}
+      aria-valuemax={max}
+      aria-valuemin={0}
       aria-valuenow={value}
       {...rest}
     >
       <div
         className={clsx(withBaseName("container"), {
           [withBaseName("indeterminate")]: variant === "indeterminate",
-          [withBaseName("static")]: variant === "static",
+          [withBaseName("determinate")]: variant === "determinate",
         })}
```

Comment:
would change to `className={clsx(withBaseName("container"), withBaseName(variant))}`